integration/tests/cook/test_basic.py

```diff
@@ -2606,3 +2606,25 @@ def predicate(job):
             self.assertEqual([1, 2], exit_codes, job)
         finally:
             util.kill_jobs(self.cook_url, [job_uuid], assert_response=False)
+
+    @unittest.skipUnless(util.using_kubernetes(), 'Test requires kubernetes')
+    def test_kubernetes_disallowed_volumes(self):
+        settings = util.settings(self.cook_url)
+        disallowed_container_paths = util.get_in(settings, 'kubernetes', 'disallowed-container-paths')
+        if disallowed_container_paths is None:
+            self.skipTest('Requires disallowed container paths')
+        path = disallowed_container_paths[0]
+        command = f'bash -c \'if [[ -e {path} ]]; then exit 1; fi\''
+        docker_image = util.docker_image()
+        container = {'type': 'docker',
+                     'docker': {'image': docker_image,
+                                'network': 'HOST',
+                                'force-pull-image': False},
+                     'volumes': [{'host-path': '/tmp',
+                                  'container-path': path}]}
+        job_uuid, resp = util.submit_job(self.cook_url, command=command, container=container)
+        self.assertEqual(201, resp.status_code)
+        try:
+            util.wait_for_instance(self.cook_url, job_uuid, status='success')
+        finally:
+            util.kill_jobs(self.cook_url, [job_uuid])
```

Comment:
This test will fail if/when GKE itself mounts the volume

Reply:
Changed the test per our discussion and added a comment.